tests/test_boa_build.py

```diff
@@ -14,16 +14,20 @@
 tests_dir = pathlib.Path(__file__).parent / "tests-v2"
 
 
-def test_build_recipes():
+def test_build_recipes(tmp_path: Path):
     recipes = [str(x) for x in recipes_dir.iterdir() if x.is_dir()]
+    cbc_yaml = tmp_path / "conda_build_config.yaml"
+    cbc_yaml.write_text("cxx_compiler:  # [win]\n- vs2022  # [win]\n")
```

Comment:
No idea where `boa` looks for `conda_build_config.yaml` files by default.
Running the tests locally with `conda_build_config.yaml` in the recipe's directory or base environment's root dir didn't seem to be picked up -- hence putting this explicitly in here.

Reply:
reason for the change: https://github.com/conda-forge/vc-feedstock/issues/78